compose/.env.schema

```diff
@@ -0,0 +1 @@
+POSTGRES_IMAGE=postgres:9.6
```

Comment:
Not sure we'll want the postgres version to change between envs... we can probably just hardcode this particular property into the docker compose to keep everyone consistent.

Reply:
sounds good - i agree - just an easy var to get this basic skeleton started